stubs/WordPress/class-wp-admin-bar.php

```diff
@@ -24,8 +24,17 @@ public function add_menu( $node ) {
 
 	/**
 	 * @param string $id
+	 * @return array{id: string, title?: string|false, parent?: string|false, href?: string|false, group?: bool, meta?: array{html?: string, class?: string, rel?: string, onclick?: string, target?: string, title?: string|false, tabindex?: string}}|void {
+	 *     Arguments for adding a node.
 	 *
-	 * @return object|null
+	 *     @type string       $id     ID of the item.
+	 *     @type string|false $title  Title of the node.
+	 *     @type string|false $parent Optional. ID of the parent node.
+	 *     @type string|false $href   Optional. Link for the item.
+	 *     @type bool         $group  Optional. Whether or not the node is a group. Default false.
+	 *     @type array        $meta   Meta data including the following keys: 'html', 'class', 'rel', 'lang', 'dir',
+	 *                                  'onclick', 'target', 'title', 'tabindex', 'menu_title'. Default empty.
+	 * }|void
```

Comment:
No void here 
```suggestion
	 * }
```

Reply:
Sorry, should've been cleared - yes, void is correct as part of the return type, but it shouldn't be here, only on line 27, where it already is :)